From 6fc03e4cf7f1a374f9e29152c38bfa0feb196066 Mon Sep 17 00:00:00 2001
From: Jernej Jakob <jernej.jakob@gmail.com>
Date: Sat, 13 Jul 2024 23:06:49 +0200
Subject: [PATCH] Fix test failure

---
 test/unit/test_autoflush.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/unit/test_autoflush.py b/test/unit/test_autoflush.py
index 2579055a..b35ad670 100644
--- a/test/unit/test_autoflush.py
+++ b/test/unit/test_autoflush.py
@@ -29,11 +29,11 @@ class TestAutoflush:
     def test_flushed(self, mock_obj):
         with Autoflush(mock_obj):
             pass
-        assert mock_obj.flush.called
+        assert mock_obj.flush.assert_called
 
     def test_transfer_err_not_flushed(self, mock_obj):
         with pytest.raises(TransferError):
             with Autoflush(mock_obj):
                 raise TransferError("bad joojoo")
-        assert mock_obj.flush.not_called
+        assert mock_obj.flush.assert_not_called
 
-- 
2.44.2

